openmdao/utils/units.py

```diff
@@ -1106,13 +1106,23 @@ def simplify_unit(old_unit_str):
     ----------
     old_unit_str : str
         Unit string to simplify.
+    msginfo : str
+        A string prepended to the ValueError which is raised if the units are invalid.
 
     Returns
     -------
     str
         Simplified unit string.
     """
-    new_str = _find_unit(old_unit_str).name()
+    if old_unit_str is None:
+        return None
+
+    found_unit = _find_unit(old_unit_str)
+    if found_unit is None:
+        _msginfo = f'{msginfo}: ' if msginfo else ''
+        raise ValueError(f'{_msginfo}The units {old_unit_str} are invalid.')
```

Comment:
I think we should put single quotes around the units in the exception, so it would look like:

The units 'in' are invalid.